src/components/table/tableToolbar/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { styled } from '@linaria/react'
+
+export const ElTableToolbar = styled.div`
+  display: flex;
+  width: 100%;
+  background: var(--fill-white);
+  padding: var(--spacing-2) 0px var(--spacing-2) 0px;
+  gap: 0px;
+  justify-content: space-between;
+  align-items: center;
+`
+
+export const ElTableToolbarDescription = styled.div`
+  font-family: var(--font-family);
+  font-size: var(--font-size-sm);
+  font-weight: 400;
```

Comment:
**issue:** I think we have CSS variables for font weights that we should be using. I can certainly see the variables defined in Figma.

Reply:
As discussed, the font-weight variables do not work and need some fixes. I will update it once the fix is in place.